### PR TITLE
Adds Space Pods (and makes earmuffs actually deafen you)

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Introducing new Nanotrasen™ Space Pods ™ ® ©
Flex your credits, AND ignore all outside distractions and criticism at the same time!

(also makes earmuffs actually deafen you but whatever)

## Why It's Good For The Game
space pods

## Changelog
:cl:
add: new Nanotrasen™ Space Pods
fix: earmuffs actually make you deaf now
/:cl:

### CODE REVIEW

**Comment by ma44** [comment]
It needs to provide ear healing and ear protection like the earmuffs do right below

File: code/modules/clothing/ears/_ears.dm [hunk 1]
```diff
@@ -7,6 +7,19 @@
 	slot_flags = ITEM_SLOT_EARS
 	resistance_flags = NONE
 
+/obj/item/clothing/ears/spacepods
+	name = "nanotrasen space pods"
+	desc = "Flex your money, AND ignore what everone else says, all at once!"
+	icon_state = "spacepods"
+	item_state = "spacepods"
+	strip_delay = 100 //air pods don't fall out
+	custom_price = 1800
```
Comment (Mickyan) [inline code/modules/clothing/ears/_ears.dm]: custom_premium_price

**Comment by carlarctg** [comment]
make it actually block all noise

**Comment by Shadowflame909** [comment]
Aw, I thought this was actual space pods and not a lame meme for earbuds

**Comment by Detective-Google** [comment]
> Aw, I thought this was actual space pods and not a lame meme for earbuds

I panicked before I realized it was a meme

**Comment by improvedname** [comment]
Make these a subtype of the earmuffs. These dont do anything else special and earmuff already use the compements your using.

**Comment by improvedname** [comment]
Currently all this do is just bowman protection

**Comment by Paxilmaniac** [comment]
there is no component for making things deaf is why

**Comment by cacogen** [comment]
add an earwax var

**Comment by trollbreeder** [comment]
You got me this time, i thought you were adding space pods (vehicle) not space pods (headphone)

**Comment by carlarctg** [comment]
that is no excuse for not adding one you LAZY CAT

**Comment by Paxilmaniac** [comment]
oh god

**Comment by Buggy123** [comment]
Just for getting my hopes up like that i'm gonna add actual spacepods.

**Comment by wesoda25** [comment]
don’t they’re shit gameplay

**Comment by cacogen** [comment]
we have to be ready for the natural progression: space stations 13

**Comment by XDTM** [comment]
all you have to do is make them add the deaf trait when equipped, and remove it when dropped, no need for super-fancy stuff

**Comment by Ohagi-Chan** [comment]
That's a terrible fix.
What if the character was otherwise deaf?

**Comment by Paxilmaniac** [comment]
then it makes them double deaf

**Comment by cacogen** [comment]
hitler is right making them physically deaf isn't a solution. think apple summer camp

**Review by spookydonut — APPROVED** [review]
Good use of traits

**Comment by XDTM** [comment]
@Ohagi-Chan the trait system is made specifically for that kind of issue, traits from separate sources are handled individually

**Review by kriskog — APPROVED** [review]

**Pull request merged**